primitives/core/src/sandbox.rs

```diff
@@ -40,6 +40,11 @@ pub enum ExternEntity {
 	Memory(u32),
 }
 
+/// The maximum number of bytes that can be allocated at one time.
+// The maximum possible allocation size was chosen rather arbitrary, 32 MiB should be enough for
+// everybody.
+pub const MAX_POSSIBLE_ALLOCATION: u32 = 33554432; // 2^25 bytes, 32 MiB
```

Comment:
Why do you put this here under sandbox?

Reply:
I was thinking sanbox was a good place to put the constraint of the execution of the wasm runtime.

But thinking again this seems wrong, the constraint is on the runtime.
The runtime must not do an allocation of more than 32MiB.

But I don't know where I should put this constraint, would putting it in primitives/core/src/lib.rs be fine ?

Reply:
Probably better than keeping it in sandbox.

Reply:
yes it is now moved in primitives/core/src/lib.rs